packages/core/src/App/Containers/Layout/header/traders-hub-onboarding.tsx

```diff
@@ -25,10 +27,17 @@ const TradersHubOnboarding = observer(() => {
                         data_testid='dt_traders_hub_onboarding_icon'
                         icon={is_dark_mode_on ? 'IcAppstoreTradingHubOnboardingDark' : 'IcAppstoreTradingHubOnboarding'}
                         size={20}
-                        onClick={() => {
-                            history.push(routes.onboarding);
-                            setIsOnboardingVisited(false);
-                        }}
+                        onClick={
+                            is_next_wallet_enabled
+                                ? () => {
+                                      localStorage.setItem('walletsOnboarding', 'started');
+                                      window.dispatchEvent(new Event('storage'));
```

Comment:
similiar to comment on the other PR - whats the use case for window.dispatchEvent, and can we avoid it? it feels very much like observer pattern

Reply:
@wojciech-deriv I need `window.dispatchEvent` because `eventListener` without event dispatching can catch `storage` event only from other browser tabs (or when I change it manually)

Reply:
Okay I see, I understand now the problem of communication between hooks-based wallets and mob-x based core package.
It really does not seem good, as it enables internals of one component to talk to the internals of other component, but I see that to avoid that, major refactors would need to be done,